tf_agents/agents/ddpg/critic_network.py

```diff
@@ -36,12 +40,24 @@ def __init__(self,
                joint_fc_layer_params=None,
                joint_dropout_layer_params=None,
                activation_fn=tf.nn.relu,
+               kernel_initializer=None,
+               batch_squash=True,
+               dtype=tf.float32,
```

Comment:
should this default to `None`, and if not set the layers use whatever dtype the preprocessing layers emitted?

same for occurrences of `dtype` elsewhere?